spec/rubocop/cop/performance/caller_locations_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+fdescribe RuboCop::Cop::Performance::CallerLocations do
```

Comment:
This look like filter option.
If are you using filter option in development, you have to remove after development.